test/test_eos.py

```diff
@@ -328,7 +339,7 @@ def inf_norm(x):
             assert inf_norm(conc[spec]) < 1e-14
 
 
-# TODO remove this test.. It is already covered in the other ones
+# FIXME This test is kinda redudant
```

Comment:
This comment goes from bad to worse.  Can you make it a little more clear to dummies?

Reply:
Essentially, we have 3 tests:
1) pyrometheus vs cantera
2) mirge-com vs pyrometheus
3) mirge-com vs cantera

My original suggestion was to drop this out. But keeping it is also fine. So either I remove the comment or the test.

Reply:
I would just leave the tests, then.  It is not entirely clear there is real redundancy here without more details.   I can see motivation for making sure that our massage of Pyro into mirgecom didn't break pyro, that mirgecom gets the same answers as pyro when using mirgecom API, and that mirgecom matches Cantera when we do a dynamic problem.

Reply:
:+1: